bow-swift/bow
# Use fastlane to create github releases

## Related issues
https://github.com/bow-swift/bow/issues/36

## Goal
This PR automates the following steps in the release process using fastlane:

- Adds fastlane setup.
- Update the `version` in `Bow.podsec`.
- Create a tag with message added in the `CHANGELOG` file.

## Implementation details
I have added a new `CHANGELOG` file. We could include any release notes into that file and our lane will automatically include that in the release note when creating the tag.

## Testing details
- Run `bundle update`
- Run `bundle exec fastlane release version_number:` with a supplied version and see the release being created along with update to `Bow.podsec` file
